.github/workflows/publish_pypi.yml

```diff
@@ -0,0 +1,31 @@
+on: push
+name: Publish Python 🐍 distributions 📦 to PyPI
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
```

Comment:
@yvkrishna Can you please confirm which python version you used to develop this code